base/applications/regedit/lang/ro-RO.rc

```diff
@@ -31,17 +31,17 @@ Opțiuni:\n\
                  Compatibil cu orice alt comutator. Ignorat.\n\
   /R:user.dat    Locația fișierului user.dat urmează să fie modificată.\n\
                  Compatibil cu orice alt comutator. Ignorat.\n\
-  /C             Importă conținutul unui fișier de regiștri.\n\
-  /D             Șterge o cheie de regiștrii specificată.\n\
-  /E             Exportă conținutul unei cheie de regiștri specificată într-un fișier.\n\
+  /C             Importă conținutul unui fișier de regiștri. Este importat întregul registru dintr-un fișier.\n\
+  /D             Șterge o cheie de registru specificată.\n\
+  /E             Exportă conținutul unei chei de registru specificată într-un fișier. Este exportat întregul registru într-un fișier.\n\
                  Dacă nu este nicio cheie specificată, toți regiștrii sunt exportați.\n\
   /S             Modul silențios. Nu sunt afișate mesaje.\n\
   /V             Lansează interfața grafică (GUI) în modul avansat. Ignorat.\n\
   /?             Afișează informații și părăsește programul.\n\
-  [filename]     Locația fișierului ce conține informații despre regiștri ce urmează\n\
+  [filename]     Este importat întregul registru dintr-un fișier. Locația fișierului ce conține informații despre regiștri ce urmează\n\
```

Comment:
The addition does not make sense. The only thing to be changed was "registru" instead of "regiștri".